website/docs/cli/cloud/command-line-arguments.html.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: "docs"
+page_title: "Command Line Arguments"
+description: "Command Line Arguments"
+---
+
+# Command Line Arguments
+
+For configurations that include a `cloud` block to integrate with Terraform Cloud, commands that
+make local modifications to Terraform state and then push them back up to the remote workspace
+accept the following option to modify that behavior:
+
+* `-ignore-remote-version` - Override checking that the local and remote
+  Terraform versions agree, making an operation proceed even when there is
+  a mismatch.
+
+    Normally state-modification operations require using a local version of
+    Terraform CLI which is compatible with the Terraform version selected
+    for the remote workspace as part of its settings. This is to avoid the
+    local operation creating a new state snapshot which the workspace's
+    remote execution environment would then be unable to decode.
+
+    Overriding this check can result in a Terraform Cloud workspace that is
+    no longer able to complete remote operations, so we recommend against
```

Comment:
I want to add something but don't know how best to wordsmith, so if this isn't the best, open to suggestions @laurapacilio !

```suggestion
    no longer able to complete remote operations with the currently selected Terraform version, so we recommend against
```

The intent is to not make it sound like the workspace is irreparably broken and will never ever complete remote operations ever again. :D